### ISSUE TITLE
A missing category in privacy spy: user info available on requirement from law agencies

### ISSUE THREAD

**Issue by zzlazz** [Politiwatch/privacyspy#128]
Apparently there is a missing category in the evaluation: what  user information can be given to law agencies, that is, what user information is the service provider able (technically) to give to law enforcement agencies upon request? (this request is called subpoena in the US but let's be generic). We should assume here that the service provider would not resist to any demands of user information by a law agency about a particular user in a law case or investigation. All the information accessible technically by the provider staff should be considered here. 
This is an acute question as for example, [here](https://www.expatica.com/ch/culture/sw-protonmail-scandal-tarnishes-swiss-privacy-reputation-220158/) protonmail, was forced to deliver info about a french activist user (the IP address and device type), which contributed to the arrest of this activist (and jailing) for a quite petty crime. On the other hand, it seems that Signal.org can only know the date of the initial account setup, and the last connection time. If the user is connected all the time, this leaves little user exposure through signal.org. 
It would also be interesting to list in what country is the provider based , therefore to which law it obeys.

**Comment by doamatto** [comment]
There is currently `law-enforcement` (see [rubric](https://privacyspy.org/contribute/)) for mentioning if data will be shared. Logically, any data a service collects that could be shared can be assumed to be shared. With your ProtonMail example, that could be things ranging from unencrypted emails, IP addresses, billing information, et al. As for what is shared, there's no real way of knowing: afaik no privacy policy states that.

> It would also be interesting to list in what country is the provider based , therefore to which law it obeys.

This is actually a pretty complicated thing. Falling back to your ProtonMail example, Proton is a Swiss company and, therefore, follows Swiss laws. However, the complexity of treaties and international alliances is the sole reason why people are weary of services in the 5/9/14 Eyes: to put in lamest terms, it allows those countries to ask each other for just about anything. Switzerland isn't an exception to this: they're part of the Club de Berne and has MLAT treaties with other countries like the [U.S.](https://2009-2017.state.gov/j/inl/rls/nrcrpt/2014/vol2/222469.htm).

tl;dr: Citing what country a service is in direct jurisdiction of doesn't seem necessarily practical considering how deep and complicated alliances and treaties are.

**Comment by doamatto** [comment]
I'm open to re-opening this in the future, but imo what I said last year holds true — it's a very nuanced topic and one that feels outside the scope of this project.
